components/slides/Preview.js

```diff
@@ -67,22 +67,18 @@ export default function Preview({ back }) {
   return (
     <div className="flex w-full flex-col items-center">
       <button
-        className="absolute left-0 m-10 opacity-80 outline-none transition-all ease-in-out hover:opacity-100"
+        className="absolute left-0 m-5 opacity-80 outline-none transition-all ease-in-out hover:opacity-100 md:m-10"
         onClick={back}
       >
         ◄ Go Back
       </button>
```

Comment:
Make this back button a re-useable component. and make it float in top left corner instead.